velox/exec/benchmarks/OrderByBenchmarkUtil.cpp

```diff
@@ -0,0 +1,308 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/benchmarks/OrderByBenchmarkUtil.h"
+#include "velox/vector/fuzzer/VectorFuzzer.h"
+#include "velox/vector/tests/utils/VectorMaker.h"
+
+DEFINE_double(data_null_ratio, 0.7, "Data null ratio");
+
+using namespace facebook::velox;
+
+namespace facebook::velox::exec {
+namespace {
+
+void addBenchmark(
+    const std::string& prefix,
+    const std::string& keyName,
+    const std::vector<vector_size_t>& batchSizes,
+    const std::vector<RowTypePtr>& rowTypes,
+    const std::vector<int>& numKeys,
+    int32_t iterations,
+    std::function<void(
+        const std::string& testName,
```

Comment:
testName is not used?

Reply:
It is used here https://github.com/facebookincubator/velox/pull/10041/files#diff-a3c00ce2041a5d08e67ea881239652bfcc9eab6eea746efc1391810332287c29R49